deno.json

```diff
@@ -13,5 +13,8 @@
   },
   "test": {
     "exclude": ["www/"]
+  },
+  "fmt": {
+    "exclude": ["docs/faq.md"]
```

Comment:
Why does this file need to be excluded from formatting?

Reply:
The autoformatting breaks markdown syntax.
I'll happily take the role of janitor for that file ;3